deps/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/deps/transport/amqp/AmqpConnectionTest.java

```diff
@@ -97,7 +97,7 @@ public void AmqpsConnectionConstructorSucceeds() throws IOException
         new NonStrictExpectations()
         {
             {
-                mockedProton.reactor((ReactorOptions)any, (Handler)any);
+                Proton.reactor((ReactorOptions)any, (Handler)any);
```

Comment:
same comment as above here and other places.

Reply:
the mocked instances might be used for other assertions in these unit tests. the only instances where we were calling static methods on these instances have been reverted.